docs/plugins.md

```diff
@@ -390,3 +400,55 @@ Leaflet keeps it simple. If you can think of a feature that is not required by a
 There are no hard requirements on how to create your own plugin, but all developers are encouraged to read the recommendations in the [plugin guide](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md).
 
 Once your plugin is ready, you can submit it: just send a pull request with a new plugin file in [/docs/_plugins/](https://github.com/Leaflet/Leaflet/tree/main/docs/_plugins)to our GitHub repository.
+
+<script>
+function loadRepoData() {
+	const rows = document.querySelectorAll('table.plugins tr');
+	rows.forEach((row) => {
+		try {
+			const repoData = row.querySelector('.repo-data');
+			if (repoData) {
+				const link = row.querySelector('.plugin-repo-url').href;
+				let badges = [];
+
+				const regexpGithubCom = /^https?:\/\/(?:www\.)?github\.com\/([\w\d-_.]+)\/([\w\d-_.]+)\/?/;
```

Comment:
The part of my brain that tells me to pursue performance where it doesn't matter at all is telling me to move the definition of the three regexps up into the context of the `loadRepoData()` function.